spec/features/layout_spec.rb

```diff
@@ -1,18 +1,38 @@
 # frozen_string_literal: true
 
 RSpec.feature 'Gov UK Layout', type: :feature do
-  scenario 'GDS styled home page' do
+  before do
     visit '/'
+  end
+
+  context 'when environment is production' do
+    around do |example|
+      with_env('production') { example.run }
+    end
+
+    scenario 'GDS styled home page' do
+      within '.govuk-header' do
+        within '.govuk-header__content' do
+          expect(page).to have_link('View court data')
+        end
+      end
 
-    within '.govuk-header' do
-      within '.govuk-header__content' do
-        expect(page).to have_link('View court data')
+      within '.govuk-phase-banner' do
+        expect(page).to have_css('.govuk-phase-banner__content', text: 'alpha')
+        expect(page).to have_css('.govuk-phase-banner__text', text: 'This is a new service')
       end
     end
+  end
+
+  context 'when environment is uat' do
```

Comment:
Should we cover the other environments as well in this test?

Reply:
👍 have added tests for the other environments